src/BenchmarksApps/DistributedCache/DistributedCache.csproj

```diff
@@ -4,16 +4,9 @@
     <OutputType>Exe</OutputType>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
-
-    <!-- Set package versions to float to latest based on TFM if not defined -->
-    <MicrosoftNETCoreAppPackageVersion Condition="'$(MicrosoftNETCoreAppPackageVersion)' == '' and '$(TargetFramework)' != ''">$(TargetFramework.Substring(3,3)).*</MicrosoftNETCoreAppPackageVersion>
-    <MicrosoftAspNetCoreAppPackageVersion Condition="'$(MicrosoftAspNetCoreAppPackageVersion)' == '' and '$(TargetFramework)' != ''">$(TargetFramework.Substring(3,3)).*</MicrosoftAspNetCoreAppPackageVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <!-- Use the latest version based on the requested TFM -->
-    <PackageReference Include="Microsoft.Extensions.Options" Version="$(MicrosoftNETCoreAppPackageVersion)" />
-    <PackageReference Include="Microsoft.Extensions.Caching.Abstractions" Version="$(MicrosoftNETCoreAppPackageVersion)" />
```

Comment:
@sebastienros - do you know why we had these 2 packages explicitly referenced? They come through the ASP.NET shared fx, so they don't need to be referenced explicitly.